File: EVA2_PRACTICA_08.java

import java.util.Scanner;

/*
 * Escribir un programa que determine si un triangulo es equilatero, isoceles o escaleno
 */

/**
 *
 * @author riguidix
 */
public class Principal {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
        int a, b, c;
        Scanner input = new Scanner(System.in);
        
        System.out.println("Introduzca el primer valor");
        a = input.nextInt();
        System.out.println("Introduzca el segundo valor");
        b = input.nextInt();
        System.out.println("Introduzca el tercer valor");
        c = input.nextInt();
        
        // Equilatero
        if (a == b && a == c ) {
            System.out.println("El triangulo es Equilatero");
        } else if (a == b && b != c || b == c && c != a || a == c && c != b) {
        // Isoceles    
            System.out.println("El triangulo es Isoceles");
        } else if (a != b && b != c && c != a) {
        // Escaleno
            System.out.println("El triangulo es escaleno");
        }        
    }   
}

File: EVA2_PRACTICA_08.py
a = int(input('Introduzca el primer valor: '))
b = int(input('Introduzca el segundo valor: '))
c = int(input('Introduzca el tercer valor: '))

if (a == b and a == c) :
    print('El triangulo es equilatero')
elif (a == b and b != c or b == c and c != a or a == c and c != b) :
    print('El triangulo es Isoceles')
elif (a != b and b != c and c != a) :
    print('El triangulo es escaleno')